catalog/USGS/NLCD_RELEASES/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS.jsonnet

```diff
@@ -0,0 +1,331 @@
+    most pixels. The structural change model provides more robust statistics on the significant 
+    minority of pixels with non-monotonic trends, while detrending some interannual signal 
+    potentially superfluous from a long-term perspective. Trends products can be downloaded from the
+    [Multi-Resolution Land Characteristics Consortium](https://www.mrlc.gov/data).
+
+    See also:
+
+    * Rigge, M., C. Homer, L. Cleeves, D. K. Meyer, B. Bunde, H. Shi, G. Xian,
+      S. Schell, and M. Bobo. 2020. Quantifying western U.S. rangelands as
+      fractional components with multi-resolution remote sensing and in situ
+      data. Remote Sensing 12.
+      [doi:10.3390/rs12030412](https://doi.org/10.3390/rs12030412)
+
+    * Rigge, M., C. Homer, H. Shi, D. Meyer, B.
+      Bunde, B. Granneman, K. Postma, P. Danielson, A. Case, and G. Xian. 2021.
+      Rangeland Fractional Components Across the Western United States
+      from 1985 to 2018. Remote Sensing 13:813.
+      [doi:10.3390/rs13040813](https://doi.org/10.3390/rs13040813)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'climate_change',
+    'disturbance',
+    'landsat_derived',
+    'nlcd',
+    'rangeland',
+    'trends statistics',
+  ],
+  providers: [
+    ee.producer_provider(
+      'United States Geological Survey and Bureau of Land Management',
+      'https://www.mrlc.gov/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(
+      -125.0683594, 28.45903302, -101.0742188, 49.32512199,
+      '1985-01-01T00:00:00Z',
+      '2021-12-31T00:00:00Z'),
+  'gee:interval': {
+    type: 'cadence',
+    unit: 'year',
+    interval: 1,
+  },
+  summaries: {
+  },
+  ],
+    gsd: [
+      30,
+    ],
+    'eo:bands': [
+      {
+        name: 'Break Point Count',
+        description: 'Number of structural breaks observed in the time-series',
+        'gee:units': 'number',
+      },
+      {
+        name: 'P-Value of Linear Model',
+        description: 'P-value of linear trends model × 100',
+        'gee:units': 'confidence',
+      },
+      {
+        name: 'Slope of Linear Model',
+        description: 'Slope of linear trends model, given in units of % change/year × 100',
+        'gee:units': 'units of percent change per year',
+      },
+      {
+        name: 'Year of Most Recent Break',
+        description: 'Year of most recent break in the time-series for each component',
+        'gee:units': 'year',
+      },
+      {
+        name: 'Total Change Intensity Index',
+        description: 'Total Change Intensity is a derivative index, designed to highlight the total amount of change across primary components (shrub, bare ground, litter, and herbaceous). Change is reflective of the slope values from the structural change analysis. Values are indexed so that the maximum observed change across all components and no change equaled 100 and 0, respectively',
+        'gee:units': 'index',
```

Comment:
This is unitless, so  eventually, that will be`1 (class)`, but for now, make is `Class`

What are the bins?

Reply:
Bins?